packages/twenty-front/src/modules/object-metadata/schemas/updateObjectMetadataItemPayloadSchema.ts

```diff
@@ -0,0 +1,31 @@
+import camelCase from 'lodash.camelcase';
+import { z, ZodTypeDef } from 'zod';
+
+import { objectMetadataItemSchema } from '@/object-metadata/schemas/objectMetadataItemSchema';
+import { UpdateObjectInput } from '~/generated-metadata/graphql';
+
+type Input = Omit<UpdateObjectInput, 'nameSingular' | 'namePlural'>;
+type Output = UpdateObjectInput;
+
+export const updateObjectMetadataItemPayloadSchema: z.ZodType<
+  Output,
+  ZodTypeDef,
+  Input
+> = objectMetadataItemSchema
+  .pick<Record<keyof Input, true>>({
+    description: true,
+    icon: true,
+    imageIdentifierFieldMetadataId: true,
+    isActive: true,
+    labelIdentifierFieldMetadataId: true,
+    labelPlural: true,
+    labelSingular: true,
+  })
+  .partial()
+  .transform((value) => ({
+    ...value,
+    nameSingular: value.labelSingular
```

Comment:
same here

Reply:
Done, this one was renamed `settingsUpdateObjectInputSchema`.